### ISSUE TITLE
`on_send` not called for `when` blocks with multiple conditionals

### ISSUE THREAD

**Issue by ptarjan** [bbatsov/rubocop#2869]
For this code

```
case foo
when 1, 2
  raise bar(3)
end
```

`on_send` will not be called for `bar(3)`. For this code

```
case foo
when 1
  raise bar(2)
end
```

it is called

**Comment by alexdowad** [comment]
What does `ruby-parse` output in either case?

**Comment by ptarjan** [comment]
@alexdowad for the first one

```
(case
  (send nil :foo)
  (when
    (int 1)
    (int 2)
    (send nil :raise
      (send nil :bar
        (int 3)))) nil)
```

for the second one

```
(case
  (send nil :foo)
  (when
    (int 1)
    (send nil :raise
      (send nil :bar
        (int 2)))) nil)
```

**Comment by alexdowad** [comment]
Aha. Nice catch.

**Comment by alexdowad** [comment]
I have a fix.

**Comment by alexdowad** [comment]
Do you have some "real life" code which you encountered this problem on? Would be good for a test case.

**Comment by ptarjan** [comment]
My real-life example boiled down pretty well to this test code. It just had longer variable names.

**Comment by alexdowad** [comment]
Hmm. That code doesn't cause an offense either way... but, whatever.

**Comment by ptarjan** [comment]
Sorry, I was building a Cop internally to remove all the `,)` from the codebase, and the `on_send` wasn't being called. Not sure if you have other cops, but maybe the multiline comma one won't have inserted a `,` into a `when` block with two args.

Here was my cop if it matters.

```
  class WeirdComma < RuboCop::Cop::Cop

    MSG = 'Stop doing a weird comma'.freeze

    def on_send(node)
      _receiver, _method_name, *args = *node
      check(node, args)
    end

    private

    def check(node, elements)
      right_paren = node.loc.end
      return unless right_paren

      comma = right_paren.begin_pos - 1
      range = Parser::Source::Range.new(node.source_range.source_buffer, comma, comma + 1)
      return unless range.source == ','

      add_offense(range, range, MSG)
    end

    def autocorrect(range)
      lambda do |corrector|
        corrector.remove(range)
      end
    end
  end
```
